examples/basic-webpack/index.jsx

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var React = require('react');
+var ReactDOM = require('react-dom');
+
+var ExampleApplication = React.createClass({
+  render: function() {
+    var elapsed = Math.round(this.props.elapsed  / 100);
+    var seconds = elapsed / 10 + (elapsed % 10 ? '' : '.0' );
+    var message = 'React has been successfully running for ' + seconds + ' seconds.';
```

Comment:
You can use template literals insteadn. With template literals, you are able to make use of the syntactic sugar making substitutions like this more readable. Check: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals